### ISSUE TITLE
Project Limit Request: snowflake-connector-python - 20 GB

### ISSUE THREAD

**Issue by sfc-gh-kdama** [pypi/support#2861]
### Project URL

https://pypi.org/project/snowflake-connector-python/

### Does this project already exist?

- [X] Yes

### New limit

20 GB (original request was 100G)

### Update issue title

- [X] I have updated the title.

### Which indexes

PyPI

### About the project

This package includes the Snowflake Connector for Python, which conforms to the Python DB API 2.0 specification: https://www.python.org/dev/peps/pep-0249/

Snowflake Documentation is available at: https://docs.snowflake.com/

Source code is also available at: https://github.com/snowflakedb/snowflake-connector-python

### How large is each release?

~420MB total

5 wheels for linux, linux_aarch64 around 25 MB for each
4 wheels for Mac64, mac_aarch64 around 15 MB for each
5 Wheels for win64 for around 15MB for each

### How frequently do you make a release?

Once a Month

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by sfc-gh-kdama** [comment]
@cmaureir  Can you help me with this request.

**Comment by cmaureir** [comment]
Hey @sfc-gh-kdama :wave: thanks for the ping.
Your request would put you in the top20 of projects https://pypi.org/stats/ due to the nature of your project and current structure, I think you still have a few options to reduce the amount of wheels and also maybe the size.
Have you consider the Limited API https://docs.python.org/3/c-api/stable.html to have 1 wheel per OS? that'll mean that you can reduce your releases to 5 wheels.

Additionally, you are currently at 10G the project limit, a jump of 10 times the original size seems to be too much. For the sake of not blocking your releases, I'd recommend an initial increase to 20G but a consideration to methods to simplify your release process with less wheels. Would that work for you?

**Comment by sfc-gh-kdama** [comment]
Hi @cmaureir  It would be ideal if we can get the increase to 100GB if possible as we look in to the options to optimize the releases to 5 wheels. If its not possible we would be good with getting to 20GB to unblock our current releases.

**Comment by sfc-gh-kdama** [comment]
hi @cmaureir  following up on our request, can we get our size limit increase earliest possible to 20 GB atleast as we are blocked on a release.

**Comment by sfc-gh-kdama** [comment]
hi @cmaureir following up on our request, can we get our size limit increase earliest possible to 20 GB atleast as we are blocked on a release.

**Comment by cmaureir** [comment]
Hey @sfc-gh-kdama :wave:
As you might know, everyone working as PyPI moderator is doing this as a hobby. We care about the Python ecosystem, and PyPI to be able to distribute more and more cool packages.

I personally tend to visit my list of issues after some days, because there are many other projects that I need to take care of as well as PyPI. Even thought I understand the urgency of releases for many projects, because even myself have been on a situation of needed a bump, it's putting a lot of pressure to drop a message every day, moreover I find it not the best approach to just copy/paste the same message over and over. I really would encourage you not to spam messages in order to get the attention of people, because for some people it could really mean a reason to just stop helping, and forget about it.

I have set the project size of `snowflake-connector-python` to 20G on PyPI, and I'd encourage you to look for ways to reduce the wheel size, by looking at the alternatives that I suggested.
